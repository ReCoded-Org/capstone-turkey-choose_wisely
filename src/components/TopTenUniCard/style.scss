@import "./../../style/variables.scss";

.top_ten_uni_card {
  background-color: $light-color;
  padding: 3rem 0 2.7rem 0;
}
