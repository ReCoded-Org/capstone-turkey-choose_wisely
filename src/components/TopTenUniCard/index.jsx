import React from "react";
import LogosSlider from "../LogosSlider/index";
import { Container } from "react-bootstrap";
import "./style.scss";
import SectionTitle from "../../components/SectionTitle";
import { useTranslation } from "react-i18next";

function TopTenUniCard() {
  const { t } = useTranslation();

  return (
    <div className="top_ten_uni_card">
      <Container>
        <SectionTitle
          title={t("home.topTen.title")}
          subText={t("home.topTen.subText")}
        />
        <LogosSlider />
      </Container>
    </div>
  );
}
export default TopTenUniCard;
