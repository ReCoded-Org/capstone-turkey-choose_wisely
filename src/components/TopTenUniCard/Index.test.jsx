import React from "react";
import TopTenUniCard from "./index";
import renderer from "react-test-renderer";

test("Snapshop test of TopTenUniCard component", () => {
  const map = renderer.create(<TopTenUniCard />);
  let tree = map.toJSON();

  expect(tree).toMatchSnapshot();
});
