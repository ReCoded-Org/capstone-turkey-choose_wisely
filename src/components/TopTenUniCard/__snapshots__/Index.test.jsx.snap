// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop test of TopTenUniCard component 1`] = `
<div
  className="top_ten_uni_card"
>
  <div
    className="container"
  >
    <div
      className="section_title"
    >
      <h1
        className="big_title"
      />
      <h2
        className="title"
      >
        home.topTen.title
      </h2>
      <p
        className="sub_text"
      >
        home.topTen.subText
      </p>
    </div>
    <div
      className="logos_slider"
    >
      <div
        className="slick-slider center slick-initialized"
      >
        
        <div
          className="slick-list"
        >
          <div
            className="slick-track"
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onMouseOver={[Function]}
            style={
              Object {
                "WebkitTransform": "translate3d(0px, 0px, 0px)",
                "WebkitTransition": "",
                "msTransform": "translateX(0px)",
                "opacity": 1,
                "transform": "translate3d(0px, 0px, 0px)",
                "transition": "",
              }
            }
          />
        </div>
        
        
      </div>
    </div>
  </div>
</div>
`;
